### ISSUE TITLE
[Bug]: show database need 27 seconds on eks(s3)

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6372]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): e23e00f1d01d43e33c509a6d239ccc38b5f93409
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

mysql> show databases;

+--------------------+
| Database           |
+--------------------+
| mo_task            |
| information_schema |
| mysql              |
| system_metrics     |
| system             |
| sbtest             |
| mo_catalog         |
+--------------------+
7 rows in set (1 min 27.44 sec)

**Comment by aressu1985** [comment]
I really do not konw how to reproduce it , just use execute this command in EKS

**Comment by sukki37** [comment]
<img width="688" alt="image" src="https://user-images.githubusercontent.com/77312370/201248236-daf36612-a474-47b5-825b-076d2c2267c2.png">
<img width="613" alt="image" src="https://user-images.githubusercontent.com/77312370/201248249-c29bc450-662e-4479-afaa-fba6761b93a2.png">

**Comment by sukki37** [comment]
As we talked about earlier, @jinqinghui will keep an eye on this issue together and help us to locate the root cause, lock competition or whatever else.

**Comment by LeftHandCold** [comment]
<img width="1604" alt="image" src="https://user-images.githubusercontent.com/14086886/201310478-6fc0ea84-0188-42bc-b5d5-288c3168be69.png">

**Comment by LeftHandCold** [comment]
[cn-dn.log.tar.gz](https://github.com/matrixorigin/matrixone/files/9988998/cn-dn.log.tar.gz)

**Comment by LeftHandCold** [comment]
The root cause is that the front end waits for a period of time to execute “show databases”, during which all cn-dn txn are completed normally, and the trace is being written to s3 during this time.

**Comment by LeftHandCold** [comment]
Steps to reproduce:
1. run eks 1cn-1dn-3log
2. Wait 1 hour
3.show databases

**Comment by LeftHandCold** [comment]
After disable trace, it did not reproduce. It should be that trace blocked the frontend end

**Comment by LeftHandCold** [comment]
Waiting for one night, it took about 1 minute for the mysql client to connect to mo the next day, and it took 7s to execute show databases

**Comment by xzxiong** [comment]
log/trace 采集的耗时在该实例中，最大值为0.16 s，当前问题 影响较小

---

> <img alt="image" width="1604" src="https://user-images.githubusercontent.com/14086886/201310478-6fc0ea84-0188-42bc-b5d5-288c3168be69.png">

补充一些细节，此处 `show databaes`  所在的 connection id 1278，只做了 `show databases` 一个操作

1. 从日志上看到（截图是从 DB里直接查询，数据请看后续），`connection 1278` 有一个 8s或 6s 周期的的停顿
<img width="2556" alt="image" src="https://user-images.githubusercontent.com/3927687/201468916-705edbc0-b6e7-431c-a15d-60534524660a.png">

2. 在 span 的数据中，可以看到，每次写文件的耗时在 1ms 左右（文件都比较小），对sql执行影响 应该是 达不到 1s的
```
select span_id, node_uuid, node_type, span_name, start_time, end_time, duration / 1e9 as sec from span_info where start_time > "2022-11-11 08:24:20.003881" and end_time < "2022-11-11 08:25:10.003881" order by start_time;
```
<img width="1548" alt="image" src="https://user-images.githubusercontent.com/3927687/201469317-fc2419b1-3c36-4fa3-a4e7-7fa2723d0060.png">

3. 从 statement_info 中可以查到，该session   确实只执行了 `show databases`，如截图。
```
-- 查询所有的 show databases 
use test_local;
select statement_id, transaction_id, session_id, user, `database`, statement, node_uuid, node_type, request_at, response_at, duration / 1e9 as sec from statement_info where statement = 'show databases' ;
-- 查看 session_id: 401c3e86-619a-11ed-902c-b2e78b5726e1 的记录
select statement_id, transaction_id, session_id, user, `database`, statement, node_uuid, node_type, request_at, duration / 1e9 as sec from statement_info where session_id = '401c3e86-619a-11ed-902c-b2e78b5726e1' ;
```
<img width="1606" alt="image" src="https://user-images.githubusercontent.com/3927687/201469460-c41c1b39-7c6b-4a09-a9ae-694fa34037a0.png">



-----
附录: 如何直接在 MO 手动创建 statement_info、log_info 、span_info
case 1) 直接load 本地数据，数据文件 dev1:/tmp/xiezexiong/tools/mo-data1.tgz,  dev1:/tmp/xiezexiong/tools/init_with_local.sh 
```
tar zxfv mo-data1.tgz -C {folder_where_mo_startup}
sh init_with_local.sh
```

case 2) 直接读取 S3的数据, 请脚本 dev1:/tmp/xiezexiong/tools/init_with_s3.sh 
```
## 执行前请修改脚本
## param for s3 data
endpoint=s3.us-west-2.amazonaws.com
region=us-west-2
bucket=bucket_name
key=I0AM0AWS0KEY0ID00000
secret="0IAM0AWS0SECRET0KEY000000000000000000000"
## as key-prefix in cn.toml
prefix="mo-data1/etl"
## want create database
database="test1"
```

**Comment by LeftHandCold** [comment]
![image](https://user-images.githubusercontent.com/14086886/201508303-ff3314c4-7b74-4a34-95ac-22c36a48d090.png)

**Comment by LeftHandCold** [comment]
[cndn.log.tar.gz](https://github.com/matrixorigin/matrixone/files/9996586/cndn.log.tar.gz)

**Comment by LeftHandCold** [comment]
Why is there Exec.Build 6s when txn communicates and executes normally?@daviszhen

**Comment by badboynt1** [comment]
i have run more than a thousand times of show databases using script. can not reporduce this problem.
maybe waiting for one night is essential?

> After disable trace,waiting for one night, it took about 1 minute for the mysql client to connect to mo the next day, and it took 7s to execute show databases

**Comment by daviszhen** [comment]
I am not working on it today.

**Comment by daviszhen** [comment]
I am not working on it today.

**Comment by yingfeng** [comment]
it has close relationship with s3 bvt timeout error which is tracked in #6209

**Comment by daviszhen** [comment]
I am not working on it today.

**Comment by daviszhen** [comment]
It is related to the #6529 .

**Comment by aressu1985** [comment]
fixed: 0.6.0-1786399（image-tag）

mysql> show databses;
ERROR 1064 (HY000): SQL parser error: You have an error in your SQL syntax; check the manual that corresponds to your MatrixOne server version for the right syntax to use. syntax error at line 1 column 13 near " databses";
mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| mo_task            |
| test02             |
| test01             |
| test03             |
| information_schema |
| mysql              |
| system_metrics     |
| system             |
| mo_catalog         |
| test               |
+--------------------+
10 rows in set (0.27 sec)
